bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/rest/api/authentication/dto/PostOAuth2Response.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.automower.internal.rest.api.authentication.dto;
+
+/**
+ * Authentication response
+ *
+ * @author Markus Pfleger - Initial contribution
+ */
+public class PostOAuth2Response {
+    private String access_token;
```

Comment:
Can you specify the SerializedName annotation and use camel case instead?
```suggestion
    private String accessToken;
```

Reply:
Yes, will do that